ast/src/analyzed/build.rs

```diff
@@ -23,3 +23,36 @@ pub fn build_binary_expr<T>(
 pub fn build_number<T, V: Into<T>>(value: V) -> Expression<T> {
     Expression::Number(value.into())
 }
+
+pub fn build_algebraic_mul<T>(
```

Comment:
Maybe not for this PR, but would be cool to implement those operators on the expression type, no?

Reply:
yeah, I'll change it here.